guides/common/modules/proc_installing-the-dhcp-infoblox-module.adoc

```diff
@@ -1,17 +1,17 @@
 [id="Installing_the_DHCP_Infoblox_Module_{context}"]
-= Installing the DHCP Infoblox module
+= Installing the DHCP Infoblox Module
 
 Use this procedure to install the DHCP Infoblox module on {SmartProxy}.
 Note that you cannot manage records in separate views.
```

Comment:
@ekohl Any ieda what this sentence means? If it's valiad, I would prefer to make it more specific: What records? And what views?

Reply:
I don't know Infoblox that well, but I get the impression this should apply to the DNS module instead. There you can only manage a single view (chosen in the installer) per Smart Proxy. And since a domain is unique, you can't install multiple Smart Proxies (one for each view) and work around it that way.

Reply:
Let's keep it as-is for now.